app/javascript/components/applications/application_form/preferences/utils.js

```diff
@@ -12,7 +12,11 @@ export const naturalKeyFromMember = (member) => {
 }
 
 export const memberNameFromPref = (p) => {
-  return `${p.application_member.first_name} ${p.application_member.last_name}`
+  if (p && p.application_member) {
```

Comment:
weird because for this application, they all had members, and they're still showing up. very confusing!